modules/core/shared/src/main/scala/lucuma/core/math/Parallax.scala

```diff
@@ -14,48 +19,48 @@ import spire.std.long._
 /**
   * Parallax stored as microarcseconds
   * Normally parallax is expressed in milliarcseconds but simbad reports them
-  * with a higher precision thus using microarcseconds gives us enough space
-  * Absolute allowed parallax values need to be less or equal thán 180 degrees
+  * with a higher precision thus using microarcseconds gives us enough space.
+  * Parallax values need to be in the interval [0°, 180°].
+  * 180° is a theoretical limit - actual astronomical values will be very small.
   */
-sealed abstract case class Parallax protected (μas: Quantity[Long, MicroArcSecond]) {
+sealed abstract case class Parallax protected (
+  μas: Quantity[Parallax.LongParallaxμas, MicroArcSecond]
```

Comment:
This will break `lucuma-catalog` but that's fine. We should cap the negative numbers there anyway

Reply:
I'm going to modify the decoder in Explore to coerce the negative numbers we receive there.